configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
I'm not actually completely sure what's the philosophy of wxGTK/Win32 port, but I think that the goal is to be closer to GTK+ than to MSW and from this point of view, neither `wxDIB` nor images in the resources should be used in it.

@kosenko Please correct me if I'm wrong.